packages/expo-camera/src/Camera.types.ts

```diff
@@ -105,6 +105,13 @@ export enum VideoQuality {
   '4:3' = '4:3',
 }
 
+export enum CameraOrientation {
```

Comment:
I think those enums should be defined as ints instead of strings 🤔 
```ts
export enum CameraOrientation {
  portrait = 1,
  portraitUpsideDown = 2,
  landscapeLeft = 3,
  landscapeRight = 4
}
```
By doing that someone can write code like this:
```ts
onResponsiveOrientationChanged={({orientation}) => {
  if (orientation === CameraOrientation.portrait) {
  	// do something 
  }
}}
```
It should be enough for that case

Reply:
Ah yes, good idea. Thanks